src/test/java/unit/uk/gov/ida/verifyserviceprovider/configuration/MetadataConfigurationWithDefaultsTest.java

```diff
@@ -1,99 +0,0 @@
-package unit.uk.gov.ida.verifyserviceprovider.configuration;
```

Comment:
These tests are now covered by the VerifyHubConfigurationFeatureTests or the MsaMetadataConfigurationTest.